diff -up dovecot-2.1.rc1/doc/example-config/conf.d/10-mail.conf.default-settings dovecot-2.1.rc1/doc/example-config/conf.d/10-mail.conf
--- dovecot-2.1.rc1/doc/example-config/conf.d/10-mail.conf.default-settings	2011-11-15 21:24:30.000000000 +0100
+++ dovecot-2.1.rc1/doc/example-config/conf.d/10-mail.conf	2011-12-14 14:46:41.808749847 +0100
@@ -277,6 +277,7 @@
 # them simultaneously.
 #mbox_read_locks = fcntl
 #mbox_write_locks = dotlock fcntl
+mbox_write_locks = fcntl
 
 # Maximum time to wait for lock (all of them) before aborting.
 #mbox_lock_timeout = 5 mins
diff -up dovecot-2.1.rc1/doc/example-config/conf.d/10-ssl.conf.default-settings dovecot-2.1.rc1/doc/example-config/conf.d/10-ssl.conf
--- dovecot-2.1.rc1/doc/example-config/conf.d/10-ssl.conf.default-settings	2011-12-14 14:48:19.967721153 +0100
+++ dovecot-2.1.rc1/doc/example-config/conf.d/10-ssl.conf	2011-12-14 14:51:10.413786253 +0100
@@ -3,7 +3,9 @@
 ##
 
 # SSL/TLS support: yes, no, required. <doc/wiki/SSL.txt>
-#ssl = yes
+# disable plain pop3 and imap, allowed are only pop3+TLS, pop3s, imap+TLS and imaps
+# plain imap and pop3 are still allowed for local connections
+ssl = required
 
 # PEM encoded X.509 SSL/TLS certificate and private key. They're opened before
 # dropping root privileges, so keep the key file unreadable by anyone but
